db/migrate/20241121222604_add_joint_account_holder_name_and_account_holder_deconstructed_fields.rb

```diff
@@ -0,0 +1,14 @@
+class AddJointAccountHolderNameAndAccountHolderDeconstructedFields < ActiveRecord::Migration[7.1]
+  def change
+    add_column :state_file_md_intakes, :has_joint_account_holder, :integer, default: 0
```

Comment:
ack i'm so sorry but we also usually have `null: false` on the enum columns. forgot to mention in the last comment 😓

Reply:
i should've caught that! ty